http-server/src/main/java/org/triplea/modules/game/listing/GameListing.java

```diff
@@ -71,18 +77,24 @@ public static GameListing build(final Jdbi jdbi, final WebSocketMessagingBus pla
                 GameListingClient.KEEP_ALIVE_SECONDS,
                 TimeUnit.SECONDS,
                 new GameTtlExpiredListener(playerMessagingBus)))
+        .connectivityCheck(new ConnectivityCheck())
         .build();
   }
 
   /** Adds a game. */
-  String postGame(final ApiKey apiKey, final LobbyGame lobbyGame) {
+  PostGameResponse postGame(final ApiKey apiKey, final LobbyGame lobbyGame) {
     final String id = UUID.randomUUID().toString();
     games.put(new GameId(apiKey, id), lobbyGame);
     playerMessagingBus.broadcastMessage(
         new LobbyGameUpdatedMessage(
             LobbyGameListing.builder().gameId(id).lobbyGame(lobbyGame).build()));
     log.info("Posted game: {}", id);
-    return id;
+
+    final boolean successfulCheck = connectivityCheck.test(getHostForGame(lobbyGame));
+    return PostGameResponse.builder()
+        .gameId(id)
```

Comment:
I think this is getting a bit confused. The API returns a "here is your game id, but by the way, I cannot connect to your game!"

We are relying on client behavior to shut down and send a disconnect message to delist, otherwise we are waiting for the keep alive to time out. 

The server should probably not rely on client side implementation that heavily, if the connectivity check fails, we'd ideally not call the post game endpoint and/or not return a game id. 

If we do not return a game-id, which arguably is more cohesive, then the keep-alive endpoint contract becomes less consistent.